Formula/arangodb.rb

```diff
@@ -15,6 +15,7 @@ class Arangodb < Formula
   depends_on "ccache" => :build
   depends_on "cmake" => :build
   depends_on "go@1.13" => :build
+  depends_on "python@3" => :build
```

Comment:
```suggestion
  depends_on "python@3.8" => :build
```

Reply:
Ah, have missed this one, thanks for catching! 👍